eppo_core/src/precomputed.rs

```diff
@@ -0,0 +1,180 @@
+use std::collections::HashMap;
+
+use rand::Rng;
+use serde::{Deserialize, Serialize};
+use serde_with::serde_as;
+
+use crate::obfuscation::{Base64Str, Md5HashedStr};
+use crate::timestamp::Timestamp;
+use crate::ufc::{Assignment, ConfigurationFormat, Environment, ValueWire, VariationType};
+use crate::Str;
+
+#[derive(Debug, Serialize, Deserialize)]
+#[serde(rename_all = "camelCase")]
+pub struct PrecomputedConfiguration {
+    pub(crate) obfuscated: serde_bool::False,
+    pub(crate) created_at: Timestamp,
+    /// `format` is always `AssignmentFormat::Precomputed`.
+    pub(crate) format: ConfigurationFormat,
+    // Environment might be missing if configuration was absent during evaluation.
+    pub(crate) environment: Option<Environment>,
+    pub(crate) flags: HashMap<String, PrecomputedAssignment>,
```

Comment:
in the SDK-SDK response, we'll include subject and subject attributes here, where there is the `obfuscated` param. cc @sameerank who does bring up a good point about these fields. Should they be obfuscated?